SU-HSDO/suhumsci
# Hsd8 306 h2 label

# READY FOR REVIEW

# Summary
- Allow the user to set a field label as an H2 tag.

# Needed By (Date)
- Asap

# Urgency
- Medium

# Steps to Test

1. checkout this branch
1. clear cache
1. go to edit a layout on a node or a node type
1. pick a field, and click to configure the "block" field
1. in the title, enter any custom title
1. in the label display setting, choose "Above"
1. ensure the lable displays in a `<div>` tag
1. edit the field again and choose "Above H2" as the label display setting
1. ensure the label now displays as an H2 tag.


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
